extensions/github-repository-search/package.json

```
@@ -29,6 +29,27 @@
       "required": false,
       "title": "GitHub Base URL",
       "description": "Optional Base URL for GitHub Enterprise (e.g. https://github.acme-inc.com/api/v3). Defaults to the public GitHub."
+    },
+    {
+      "name": "searchInOrgs",
+      "type": "textfield",
+      "required": false,
+      "title": "Organizations / Users to search in",
+      "description": "Comma separated list of organizations or users to search in. Leave empty to search in all repositories. Use @me to search in your own repositories."
```

I assume this limts the search to only the users/orgs? Maybe it's good to clarify this in the description.
```suggestion
      "description": "Limit the search so a comma-separated list of users or organizations, e.g. 'mheidinger, raycast'. Tip: Use '@me' to search only your own repositories."
```

---

Took over the suggestion